File: 프로그래머스/unrated/181902. 문자 개수 세기/문자 개수 세기.js
function solution(my_string) {
    const answer = new Array(52).fill(0);
    
    for (let i = 0; i < my_string.length; i++) {
        if (my_string[i].toUpperCase() === my_string[i]) {
            answer[my_string.charCodeAt(i) - 65]++;
        } else {
            answer[my_string.charCodeAt(i) - 71]++;
        }
    }
    
    return answer;
}

File: 프로그래머스/unrated/181902. 문자 개수 세기/문자 개수 세기.java
class Solution {
    public int[] solution(String my_string) {
        int[] result = new int[52];
        
        for (int i = 0; i < my_string.length(); i++) {
            int code = (int) my_string.charAt(i);
            if (code <= 90) {
                code -= 65;
            } else {
                code -= 71;
            }
            result[code]++;
        }
        
        return result;
    }
}